Parsers/weekMoji.js

```diff
@@ -0,0 +1,21 @@
+/*
+emoji: saturday-chill
```

Comment:
This trigger is not a valid emoji. Thank you for making the other changes. Maybe the calendar emoji would be a good trigger?

Reply:
Made the changes in header. Thanks